src/utils/duration.js

```diff
@@ -0,0 +1,63 @@
+const ONE_MINUTE = 60 * 1000;
```

Comment:
Open MCT Copyright stuff if necessary.

and... Cool! :)

Reply:
Done.